Update help docs for SPSE (#72)
MicrosoftDocs/OfficeDocs-SharePoint-PowerShell: sharepoint/sharepoint-server-ps/sharepoint-server/New-SPWebApplication.md

### ------------------EXAMPLE 1-----------------------
```powershell

> New-SPWebApplication -Name "Contoso Internet Site" -Port 80 -HostHeader sharepoint.contoso.com -URL "https://www.contoso.com" -ApplicationPool "ContosoAppPool" -ApplicationPoolAccount (Get-SPManagedAccount "DOMAIN\jdoe")

Reviewer: Being that the port specified is 80, is this still https in the URL param?

Reviewer 2: "DOMAIN\wa" would be preferred rather than 'jdoe' as that sounds like an end user name vs. a service account.

Reviewer 3: FWIW, I think "wa" is too short to be clear as a service account name.  Should we use "serviceaccount" instead?

Reviewer 2: @TroyStarr I think we may use it in other 'best practice' documentation for SP Svr but would need to look it up. We signify 'wa' for 'WebApplication' and 'sa' for 'Service Application'.

Reviewer 3: Unless extreme compactness is needed, I recommend a longer example account name to give a better hint of what the purpose of the account is. :-)

New-SPWebApplication -Name "Contoso Internet Site" -Port 80 -HostHeader sharepoint.contoso.com -URL "http://www.contoso.com" -ApplicationPool "ContosoAppPool" -ApplicationPoolAccount (Get-SPManagedAccount "DOMAIN\wa")
```

> This example creates a new Web application by using an internal host header of sharepoint.contoso.com and a public URL of https://www.contoso.com.

Reviewer 4: ```suggestion
This example creates a new web application by using an internal host header of sharepoint.contoso.com and a public URL of https://www.contoso.com.
```

Reviewer 3: The example and its description need to match, so if we're updating the description to use an SSL URL, then the example needs to be updated too.  But the example is also using port 80, which is an HTTP port rather than an HTTPS port.

## PARAMETERS

### -AdditionalClaimProvider

> Adds a specific claim provider to the defined Web application.

Reviewer 4: ```suggestion
Adds a specific claim provider to the defined web application.
```

```yaml
Type: SPClaimProviderPipeBind[]
Parameter Sets: (All)
Aliases: 
Applicable: SharePoint Server Subscription Edition

Required: False
Position: Named
Default value: None
Accept pipeline input: False
Accept wildcard characters: False
```

### -AllowAnonymousAccess

> Allows anonymous access to the Web application.

Reviewer 4: ```suggestion
Allows anonymous access to the web application.
```

```yaml
Type: SwitchParameter
Parameter Sets: (All)
Aliases: 
Applicable: SharePoint Server Subscription Edition

Required: False
Position: Named
Default value: None
Accept pipeline input: False
Accept wildcard characters: False
```

### -AllowLegacyEncryption

> Specifies that older SSL and TLS protocol versions and cipher suites are allowed to be used with this IIS web site.

Reviewer 4: ```suggestion
Specifies that older SSL and TLS protocol versions and cipher suites are allowed to be used with this IIS website.
```

Accept wildcard characters: False
```

### -Certificate

> Specifies the certificate that will be used for the Secure Sockets Layer (SSL) binding of this IIS web site.

Reviewer 4: ```suggestion
Specifies the certificate that will be used for the Secure Sockets Layer (SSL) binding of this IIS website.
```

```

### -HostHeader
Specifies the host header binding for this IIS web site.

> A host header binding allows multiple IIS web sites to share the same port number.

Reviewer 4: ```suggestion
A host header binding allows multiple IIS websites to share the same port number.
```

> Web requests sent to a shared port number are routed to the correct IIS web site based on the value of the HTTP host header sent by the client.

Reviewer 4: ```suggestion
Web requests sent to a shared port number are routed to the correct IIS website based on the value of the HTTP host header sent by the client.
```

> If no host header binding is specified, then all web requests sent to this port number will be routed to this IIS web site unless another IIS web site has a host header binding that matches the HTTP host header sent by the client.

Reviewer 4: ```suggestion
If no host header binding is specified, then all web requests sent to this port number will be routed to this IIS website unless another IIS website has a host header binding that matches the HTTP host header sent by the client.
```

Accept wildcard characters: False
```

### -Name

> Specifies the name of the new Web application.

Reviewer 4: ```suggestion
Specifies the name of the new web application.
```

```yaml
Type: String
Parameter Sets: (All)
Aliases: 
Applicable: SharePoint Server Subscription Edition

Required: True
Position: Named
Default value: None
Accept pipeline input: True (ByValue)
Accept wildcard characters: False
```

### -Path

> Specifies the physical directory for the new Web application in the virtual directories folder.

Reviewer 4: ```suggestion
Specifies the physical directory for the new web application in the virtual directories folder.
```

### -SecureSocketsLayer
Enables Secure Sockets Layer (SSL) encryption for this Web application.

> If you choose to use SSL, you must import a server certificate to SharePoint and assign it to the IIS web site for this web application.

Reviewer 4: ```suggestion
If you choose to use SSL, you must import a server certificate to SharePoint and assign it to the IIS website for this web application.
```

> Until this is done, the Web application will be inaccessible from this IIS Web site.

Reviewer 4: ```suggestion
Until this is done, the Web application will be inaccessible from this IIS website.
```

The default value is False.

> If this parameter is omitted or set to False, this Web application will use HTTP for the specified port.

Reviewer 4: ```suggestion
If this parameter is omitted or set to False, this web application will use HTTP for the specified port.
```

Accept wildcard characters: False
```

### -UseServerNameIndication

> Specifies that the Secure Sockets Layer (SSL) binding of this IIS web site should use Server Name Indication (SNI).

Reviewer 4: ```suggestion
Specifies that the Secure Sockets Layer (SSL) binding of this IIS website should use Server Name Indication (SNI).
```

> Server Name Indication allows multiple IIS web sites with unique host headers and unique server certificates to share the same SSL port.

Reviewer 4: ```suggestion
Server Name Indication allows multiple IIS websites with unique host headers and unique server certificates to share the same SSL port.
```

> If Server Name Indication isn't used, all IIS web sites sharing the same SSL port must share the same server certificate.

Reviewer 4: ```suggestion
If Server Name Indication isn't used, all IIS websites sharing the same SSL port must share the same server certificate.
```
